.tests/test-workflow/config/cancer_wes/config.yaml.jinja2

```diff
@@ -0,0 +1,63 @@
+static_data_config:
+  reference:
+    path: {{ params["reference"] }}
+
+step_config:
+  ngs_mapping:
+    tools:
+      dna: [bwa]    # Required if DNA analysis; otherwise, leave empty. Example: 'bwa'.
+    bwa:
+      path_index: {{ params["bwa_index"] }}
+      mask_duplicates: false
+      memory_bam_sort: 2G
+      num_threads_bam_sort: 2
+      num_threads_bam_view: 2
+      num_threads_align: 2
+    ngs_chew_fingerprint:
+      enabled: false
+    target_coverage_report:
+      path_target_interval_list_mapping:
+      - name: MedExome_hg19_empirical_targets
+        pattern: MedExome
+        path: ../resources/Exome-MedExome.chr12.bed
+
+
+  somatic_variant_calling:
+    tools: [mutect2]
+    mutect2:
+      ngs_mapping: ../ngs_mapping
+      extra_arguments: [] # [ "--genotype-germline-sites true", "--genotype-pon-sites true" ]
```

Comment:
Please remove comment

FYI: these arguments are required to generate germline & somatic variant calling when preparing for `PureCN` somatic CNV calling. I haven't found a way to recapitulate by some filtration magic the same somatic variants generated without these arguments.

Reply:
Done (and thank you for the explanation)